mautic/mautic
# Support to Pass MessagingServiceSID to Twillio API instead of From

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | No
| New feature/enhancement? (use the a.x branch)      | Yes
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | Yes 
| Related user documentation PR URL      | https://github.com/mautic/user-documentation/pull/184
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://github.com/mautic/mautic/issues/12479



#### Description:

Send messages in twillio plugin using MessagingServiceSID as suggested by Twillio.
Now we have replaced Sending Phone Number by MessagingServiceSID



#### Steps to test this PR:

1. Configure twillio plugin
2. In feature tab -> add MessagingServiceSID provided by twillio
3. verify send sms service
